string str
float64 intensity
---
MotorPos[] motor_positions
